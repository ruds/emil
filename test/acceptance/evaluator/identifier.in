true;
it;
"hi there";
it;
it : string;
