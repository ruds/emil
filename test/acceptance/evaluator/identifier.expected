val it = true : bool
val it = true : bool
val it = "hi there" : string
val it = "hi there" : string
val it = "hi there" : string
